components/meta-packages/libjpeg/Makefile

```diff
@@ -15,7 +15,7 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		libjpeg
-COMPONENT_VERSION=	6.0.2
+COMPONENT_VERSION=	8.3.2
 COMPONENT_SUMMARY=	Joint Photographic Experts Group library
 
```

Comment:
Shouldn't we enhance the summary?

Reply:
Sure, I commited what I could come up with. Open to suggestions though.